lib/rucio/core/rule.py

```diff
@@ -2242,7 +2238,7 @@ def insert_rule_history(rule, recent=True, longterm=False, *, session: "Session"
 
 
 @transactional_session
-def approve_rule(rule_id, approver=None, notify_approvers=True, *, session: "Session"):
+def approve_rule(rule_id: str, approver: Optional[str] = None, notify_approvers: bool = True, *, session: "Session") -> None:
```

Comment:
*NOT FOR 34.0.0rc2:* Does `approver: Optional[str] = None` offer any benefit against `approver: str = ''`? Same for `deny_rule()` below.